charts/backstage/values.yaml

```diff
@@ -12,8 +16,12 @@ upstream:
     image:
       registry: quay.io
       repository: janus-idp/backstage-showcase
-      tag: latest
+      tag: next
```

Comment:
please default to `latest`

Reply:
Done, but afaict that will the CI tests. Should I temporarily change the tags in [CI default values](git push origin dynamic-backend-plugins:dynamic-backend-plugins) ?

Reply:
In fact I have temporarily changed the CI test `initContainer` image tag to `next` in commit c20c21b66d1df465ba2a079834f562f16aa2f82d.
With that tests should succeed.